VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/custom/CustomSelectViewAction.java

```diff
@@ -0,0 +1,38 @@
+package org.edx.mobile.test.feature.custom;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.view.View;
+
+import org.edx.mobile.view.custom.CustomSelectView;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static org.hamcrest.Matchers.allOf;
+
+/**
+ * Defines the custom {@link ViewAction} implementation to select an item
+ * from our custom implementation of spinner i.e. {@link CustomSelectView}.
+ */
+public class CustomSelectViewAction {
```

Comment:
It's just two lines 😜 and the same code should work with our existing widget too, since it's just based on clicks and list view scrolls. It's also much more robust, won't break if we later change the widget to a standard `Spinner`, and removes the need for the custom selection method and action you've defined here (which looks like the simpler course of action).